tests/scripts/create-dev-cluster.sh

```diff
@@ -56,21 +70,31 @@ show_info() {
     echo " "
     echo " *** To start using your rook cluster please set the following alias: "
     echo " "
-    echo "    > " alias kubectl=\"$KUBECTL\"
+    echo "    >  alias kubectl=\"$KUBECTL\" "
 }
 
-check_minikube_exists() {
-    if minikube profile list | grep -q $ROOK_PROFILE_NAME; then
-        echo "A minikube profile '$ROOK_PROFILE_NAME' already exists, please use -f to force the cluster creation."
-	exit 1
+check_minikube_profile_exists() {
+    
+	minikube profile list | grep -q $ROOK_PROFILE_NAME
+	local retcode=$?
+	if [ $retcode -eq 0 ]; then
+        echo "A minikube profile '$ROOK_PROFILE_NAME' already exists."
+	echo "Using the existing minikube environment."
+	echo " please use -f to force the minikube  cluster creation."
+	return  1
+    else
+	echo " A minikube environment does not exist yet."
+	echo " please use -f to force the minikube  cluster creation."
+
+
     fi
 }
 
 setup_minikube_env() {
-    minikube_driver="$(get_minikube_driver)"
-    echo "Setting up minikube env (using $minikube_driver driver)"
+	minikube_driver="$(get_minikube_driver)"
+    echo "Setting up minikube env \(using driver ${minikube_driver}\)"
```

Comment:
There's no need to use the grouping `{}`  in the variable is simple.

Reply:
@rkachach  wrote:
> There's no need to use the grouping `{}` in the variable is simple.

I just find it more readable. No other reason.